examples/sprites/src/lib.rs

```diff
@@ -31,8 +31,8 @@ pub fn translate(p: &Point, v: Vector) -> Point {
 // and which can move about over time.
 #[derive(Debug)]
 pub struct Sprite {
-    // All interfaces must be threadsafe, meaning `Send+Sync`, and our naive
-    // `Point` struct isn't - so we wrap it in a `RwLock` to make it so.
+    // All interfaces must be `Send + Sync`, and our naive `Point` struct
```

Comment:
nit: almost everywhere else has no space between Send and Sync. Not sure which one is actually preferred, but we might as well be consistent.